src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
Please stick to encapsulation principle of oop rules, which means we need to hide our varaible/data and expose only needed information. In this situation, you didnt declare any access modifier, which means it will be a package-private, we need to lower the access level to private. Please always rember about this rule.
```suggestion
    private String ballColor;
```